pkg/kubelet/apis/kubeletconfig/validation/validation.go

```diff
@@ -41,6 +42,60 @@ func ValidateKubeletConfiguration(kc *kubeletconfig.KubeletConfiguration) error
 	if kc.SystemCgroups != "" && kc.CgroupRoot == "" {
 		return fmt.Errorf("invalid configuration: system container was specified and cgroup root was not specified")
 	}
+	if utilvalidation.IsValidPortNum(int(kc.CAdvisorPort)) != nil {
+		return fmt.Errorf("Invalid configuration: --cadvisor-port %v must be between 1 and 65535", kc.CAdvisorPort)
+	}
+	if kc.EventBurst < 0 {
+		return fmt.Errorf("Invalid configuration: --event-burst %v should not be a negative number", kc.EventBurst)
```

Comment:
nit: s/should/must
see [rfc2119](https://www.ietf.org/rfc/rfc2119.txt)
and similarly for other instances where "should" actually means "must," according to this validation code.